data/scenarios/Tutorials/debug.yaml

```diff
@@ -4,23 +4,26 @@ description: |
   Learn how to view built robots and debug them.
 objectives:
   - goal:
-      - Before you send your robots far away from the `base` you need to learn how to figure out what went wrong with them if they crash.
       - |
-        In this challenge, you should start by
-        sending a robot to walk four steps straight east into the mountain,
-        crashing deliberately.  However, you must make sure it has a `logger`{=entity},
-        so we can see what command failed.  The simplest way to ensure
+        Before you send your robots far away from the `base` you need to learn how
+        to figure out what went wrong with them if they crash.
```

Comment:
@byorgey I am considering adding an objective to salvage a built robot in the same space first.
I worry that this scenario teaches too much at once.

What do you think?

Reply:
Sure, I think that makes sense.